GitVersion.yml

```diff
@@ -2,6 +2,4 @@ assembly-versioning-scheme: Major
 next-version: 6.0
 branches:
   develop:
-    tag: alpha
-  releases?[/-]:
```

Comment:
Setting this in V4 is still supported, they just changed the syntax. Each type of branch now has a group name, and and the regex is a property of it.

See https://github.com/Particular/NServiceBus.RabbitMQ/blob/develop/GitVersion.yml for how to do this in V4.